src/index.js

```diff
@@ -0,0 +1,38 @@
+import './styles.css';
+// import checkBoxing from './check_box.js'
+// Populating the html element
```

Comment:
- [ ] I would recommend removing the commented code to make the codebase cleaner.